docs/faq.md

```diff
@@ -1,73 +1,119 @@
+
 I've configured and used wp-browser on Docker, Vagrant, [VVV](https://github.com/Varying-Vagrant-Vagrants/VVV), [Valet](https://laravel.com/docs/5.7/valet) and various CI solutions.  
+
 To this day I keep using different setups on different machines and personally prefer [Docker](https://www.docker.com/) for its portability.
 
 ### Can I only test plugins with wp-browser?
+
 No, you can test any kind of WordPress application.  
+
 With "application" I mean any PHP software built on top of WordPress: plugins, themes, whole sites.
 
 ### If I'm testing a site do I have to use the default WordPress file structure?
+
 No, you can use any file structure you want.  
+
 Some wp-browser modules will need a little help to find your code but, so far, I've never been unable to set it up.
 
 ### Can I use wp-browser even if my WordPress application doesn't use Composer?
+
 Yes, although wp-browser, as a development tool, cannot be installed without [Composer](https://getcomposer.org/).
 
 ### Should I use wp-browser to test my production servers?
+
 No. Unless you know very well what you're doing that's a dangerous idea that might leave you with a broken site and an empty database.  
+
 As almost any testing tool wp-browser should be used locally on local installations of WordPress that do not contain any valuable information.
 
 ### How can I avoid the WPDb module from replacing the contents of my database?
+
 You should always backup any database that contains any information you care about **before** running any test.  
+
 You can read the answer to this question [in the WPDb module documentation](modules/WPDb.md#change-the-databse-used-depending-on-whether-you're-running-tests-or-not).  
 
 ### Can I run all my tests with one command?
+
 Theoretically: yes, in practice: no.  
+
 When you use `codecept run` Codeception will run all the tests from all the suites.  
+
 This, done in the context of other frameworks, will generally not create any problem but, in the context of WordPress it will.  
+
 While handling a single HTTP request WordPress will set, and use, a number of constants and globals and, likewise, will do plugins and themes that follow WordPress standards.  
+
 This means that the global context (variable scope) will be left "dirty" and contain "left-over" constants and globals from the previous tests.  
+
 An example is one where a test for the handling of Ajax requests sets the `DOING_AJAX` constant: this will be now set for **any** test after the one that set it thus breaking, or worse altering, all the following ones.
+
 So, in short, **run each suite separately**.
 
 ### Can I have more than one suite of one kind?
+
 Yes, you should.  
-As an example you might have a `frontend` suite running [acceptance tests](levels-of-testing.md#acceptance-tests) on the site frontend and a `backend` suite running acceptance tests on the site backend.  
+
+As an example you might have a `frontend` suite running [acceptance tests]
```

Comment:
Hard-wrap here and not elsewhere? On purpose? Why?

Reply:
Intentional, explained on final comment.